style.css

```diff
@@ -67,3 +67,11 @@ aside {
   float: left;
 }
 
+blockquote {
```

Comment:
sample of a comment in a PR, it's pretty useful when working with teammates on the same project and to have same standard/quality in code